taxonomy-api/taxonomy-actors/src/test/resources/application.conf

```diff
@@ -0,0 +1,506 @@
+search.fields.date=["lastUpdatedOn","createdOn","versionDate","lastSubmittedOn","lastPublishedOn"]
+search.batch.size=500
+search.connection.timeout=30
+platform-api-url="http://localhost:8080/language-service"
+MAX_ITERATION_COUNT_FOR_SAMZA_JOB=2
+
+
+# DIAL Code Configuration
+dialcode.keyspace.name="dialcode_store"
+dialcode.keyspace.table="dial_code"
+dialcode.max_count=1000
+
+# System Configuration
+system.config.keyspace.name="dialcode_store"
+system.config.table="system_config"
+
+#Publisher Configuration
+publisher.keyspace.name="dialcode_store"
+publisher.keyspace.table="publisher"
+
+#DIAL Code Generator Configuration
+dialcode.strip.chars="0"
+dialcode.length=6.0
+dialcode.large.prime_number=1679979167
+
+#DIAL Code ElasticSearch Configuration
+dialcode.index=true
+dialcode.object_type="DialCode"
+
+framework.max_term_creation_limit=200
+
+# Enable Suggested Framework in Get Channel API.
+channel.fetch.suggested_frameworks=true
+
+# Kafka configuration details
+kafka.topics.instruction="local.learning.job.request"
+kafka.urls="localhost:9092"
+
+#Youtube Standard Licence Validation
+learning.content.youtube.validate.license=true
+learning.content.youtube.application.name=fetch-youtube-license
+youtube.license.regex.pattern=["\\?vi?=([^&]*)", "watch\\?.*v=([^&]*)", "(?:embed|vi?)/([^/?]*)","^([A-Za-z0-9\\-\\_]*)"]
+
+#Top N Config for Search Telemetry
+telemetry_env=dev
+telemetry.search.topn=5
+
+installation.id=ekstep
+
+
+channel.default="in.ekstep"
+
+# DialCode Link API Config
+learning.content.link_dialcode_validation=true
+dialcode.api.search.url="http://localhost:8080/learning-service/v3/dialcode/search"
+dialcode.api.authorization=auth_key
+
+# Language-Code Configuration
+platform.language.codes=["as","bn","en","gu","hi","hoc","jun","ka","mai","mr","unx","or","san","sat","ta","te","urd", "pj"]
+
+# Kafka send event to topic enable
+kafka.topic.send.enable=false
+
+learning.valid_license=["creativeCommon"]
+learning.service_provider=["youtube"]
+
+stream.mime.type=video/mp4
+compositesearch.index.name="compositesearch"
+
+hierarchy.keyspace.name=hierarchy_store
+content.hierarchy.table=content_hierarchy
+framework.hierarchy.table=framework_hierarchy
+
+# Kafka topic for definition update event.
+kafka.topic.system.command="dev.system.command"
+
+learning.reserve_dialcode.content_type=["TextBook"]
+# restrict.metadata.objectTypes=["Content", "ContentImage", "AssessmentItem", "Channel", "Framework", "Category", "CategoryInstance", "Term"]
+
+#restrict.metadata.objectTypes="Content,ContentImage"
+
+publish.collection.fullecar.disable=true
+
+# Consistency Level for Multi Node Cassandra cluster
+cassandra.lp.consistency.level=QUORUM
+
+
+
+
+content.nested.fields="badgeAssertions,targets,badgeAssociations"
+
+content.cache.ttl=86400
+content.cache.enable=false
+collection.cache.enable=false
```

Comment:
why are we adding content related configuration in taxonomy-service? Please remove all content related config which is not required in this service.

Reply:
Done